lua/nvim-treesitter/install.lua

```diff
@@ -47,7 +47,7 @@ local function iter_cmd_sync(cmd_list)
       print(cmd.info)
     end
 
-    vim.fn.system(get_command(cmd))
+    print(vim.fn.system(vim.fn.escape(get_command(cmd), '\\')))
```

Comment:
Maybe leave this print here? Otherwise, users will never see the output of the command. Could also be useful for finding errors when `:TSInstall` fails...